src/Riok.Mapperly/Descriptors/MappingBuilders/DictionaryMappingBuilder.cs

```diff
@@ -13,6 +13,8 @@ public static class DictionaryMappingBuilder
 {
     private const string SetterIndexerPropertyName = "set_Item";
 
+    private const string ToDictionaryName = "ToDictionary";
+    private const string ToDictionaryMethodName = "global::System.Collections.Generic.Enumerable.ToDictionary";
```

Comment:
Both are method names aren't they? I think names `ToDictionaryMethodName` and `ToDictionaryMethodFullName` would make things clearer, wdyt?